src/api/org.ts

```diff
@@ -272,6 +272,38 @@ export function disallowOrgToSetupSamlConnection(
     )
 }
 
+export type CreateSAMLConnectionLinkResponse = {
+    url: string
+}
+
+export function createOrgSamlConnectionLink(
+    authUrl: URL,
+    integrationApiKey: string,
+    orgId: string,
+    expiresInSeconds?: number
+): Promise<CreateSAMLConnectionLinkResponse> {
+    if (!isValidId(orgId)) {
+        return Promise.reject("Invalid orgId")
+    }
+
+    let endpointPath = `${ENDPOINT_PATH}/${orgId}/create_saml_connection_link`
+    if (expiresInSeconds) {
```

Comment:
This should actually be a JSON body parameter, instead of a query parameters.

Reply:
Ah, I was confused by the naming of the variable in the backend 😅 you're right though, changing now!

![Screenshot 2023-10-02 at 5 04 09 PM](https://github.com/PropelAuth/node/assets/36497022/96c5514b-54f4-435f-bb9b-b26b0359b439)

Reply:
great catch!